app/presenters/brexit_taxons_presenter.rb

```diff
@@ -1,28 +1,30 @@
 class BrexitTaxonsPresenter
   CITIZEN_TAXON_CONTENT_ID = 'd7bdaee2-8ea5-460e-b00d-6e9382eb6b61'.freeze
 
+  FEATURED_LINKS = %w(
+    /visit-europe-brexit
+  ).freeze
+
   FEATURED_TAXONS = %w(
-    /going-and-being-abroad
     /work
-    /transport
     /environment
     /business-and-industry
     /education
   ).freeze
 
   REJECTED_TAXONS = %w(
+    /going-and-being-abroad
     /government/all
+    /transport
   ).freeze
 
-  def featured_taxons
-    @featured_taxons ||= FEATURED_TAXONS.map.with_index do |base_path, index|
-      featured_taxon = level_one_taxons.detect { |taxon| taxon.base_path == base_path }
-      BrexitTaxonPresenter.new(featured_taxon, index)
-    end
+  def featured_links
+    custom_links +
+      featured_taxons(custom_links.length)
```

Comment:
Is there a way that you could instantiate all of the presenters here? It would be a bit more code I suppose, but it would make the index stuff a bit clearer.